win32ss/drivers/miniport/xboxvmp/xboxvmp.h

```diff
@@ -11,48 +11,32 @@
 
 /* INCLUDES *******************************************************************/
 
+/* FIXME: specify headers properly! */
```

Comment:
I'm not sure what this means...

Reply:
All headers here specified with double quotes instead of triangle ones. I experimented with different HAL and MM functions here while developing, and it was a pain to add new needed headers, almost always breaking a build.